test/integration_test.py

```diff
@@ -714,3 +724,52 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    host = '127.0.0.1'
+    port = 8000
+    url = furl(scheme='http', host=host, port=port)
+    local_chalice_server = None
+    server_thread = None
+
+    @classmethod
+    def setUpClass(cls, config=config) -> None:
+        super().setUpClass()
+        service_app = Path(config.project_root) / 'lambdas' / 'service'
```

Comment:
You should be able to derive this from `service.__path__`

Reply:
`__path__` was an invalid attribute to `service`, however `__file__` did work. I had to modify it a bit so that `factory.CLIFactory()` got what was expecting.